public/Test-AdfLinkedService.ps1

```diff
@@ -9,27 +9,37 @@ function Test-AdfLinkedService {
         [String] $ResourceGroupName,
         [parameter(Mandatory = $true)] 
         [String] $SubscriptionID,
```

Comment:
Using this opportunity, could you pass $SubscriptionID to calling functions appropriately?